doc/mediasdk-man.md

```diff
@@ -8447,6 +8486,8 @@ If the acceleration device is not default device, use the following values to in
 `MFX_IMPL_HARDWARE2` | Use the hardware-accelerated implementation on the second acceleration device.
 `MFX_IMPL_HARDWARE3` | Use the hardware-accelerated implementation on the third acceleration device.
 `MFX_IMPL_HARDWARE4` | Use the hardware-accelerated implementation on the fourth acceleration device.
+`MFX_IMPL_SINGLE_THREAD` |  Prohibits creation of internal threads. Do not create its own threads in MediaSDK library and plugin and work only in application threads.
```

Comment:
Can you, please move this out in a separate commit preceding current one? That's separate feature from partial bitstream output.

Reply:
fixed in new update of pull request